src/Gluon/paket.template

```diff
@@ -5,7 +5,7 @@ owners
     ctaggart
     panesofglass
     t0yv0
-copyright Copyright Tachyus Corp. All rights reserved.
+copyright Copyright Tachyus Corp.
```

Comment:
This text is incorrect according to the license, Apache 2.0. Non of the source code says `All Rights Reserved` since that would nullify the `Apache 2.0`